lib/mix/lib/mix/tasks/test.ex

```diff
@@ -2,6 +2,8 @@ defmodule Mix.Tasks.Test do
   defmodule Cover do
     @moduledoc false
 
+    @threshold 90
```

Comment:
This would need to be made configurable.

Reply:
This is configurable via `threshold` value in `test_coverage` configuration option. I haven't documented it yet thought.

Reply:
Beautiful. Maybe we should call this `@default_threshold` then?

Reply:
No problem